src/webnn_native/openvino/ienn/src/ie_model.cpp

```diff
@@ -438,6 +438,21 @@ ie_operand_t* Model::AddRelu(ie_operand_t* input) {
   return CreateOperand(node_name);
 }
 
+ie_operand_t* Model::AddReduceMean(ie_operand_t* input,
+                                   ie_reduce_mean_options_t* options) {
+  auto input_node = name_node_map_[input->name];
+  SizeVector axes = ToVector(options->axes, options->axesCount);
+  auto axes_node = op::Constant::create(element::i64, Shape{axes.size()}, axes);
+  // Issue: When input rank == 3 and keeDimensions == false, reduceMean produces
+  // a result with wrong shape. This will be fixed by
+  // https://github.com/openvinotoolkit/openvino/pull/5986 in OpenVINO 2021/4.
```

Comment:
@mingmingtasd , could you please file an issue about this? So later we can upgrade to 2021/4 to fix this issue. Thanks!

Reply:
Done, thanks! @huningxin

Reply:
Could you please share the link? I mean https://github.com/webmachinelearning/webnn-native/issues

Reply:
Yes, I have raised two issues in this repo, #18 for IE and #19 for DML. Thanks! @huningxin

Reply:
cool